dev_tools/nyaml2nxdl/README.md

```diff
@@ -0,0 +1,68 @@
+# YAML to NXDL converter and NXDL to YAML converter
+
+**NOTE: Please use python3.8 or above to run this converter**
+
+**Tools purpose**: Offer a simple YAML-based schema and a XML-based schema to describe NeXus instances. These can be NeXus application definitions or classes
```

Comment:
This introduces new terminology, making a distinction that does not exist between application definitions, base classes, and a mis-named contributed classes.  Please use the terminology as described in the [NeXus Class Definitions](https://manual.nexusformat.org/classes/index.html#nexus-class-definitions) section of the manual.

Reply:
What is a *NeXus instance* and how does it differ from a NeXus class?

Reply:
Thank you for pointing out the inconsistency. The language has been corrected now.